src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java

```diff
@@ -0,0 +1,44 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import java.util.Iterator;
+import static me.totalfreedom.totalfreedommod.command.Command_tag.FORBIDDEN_WORDS;
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Gives you a rainbow tag", usage = "/<command> <tag>", aliases = "tn")

```

Comment:
Do we want to be alisaing commands? As I recall we wanted to avoid it to keep things simple?